near-sdk/src/environment/env.rs

```diff
@@ -80,6 +77,7 @@ macro_rules! method_into_register {
 ///       );
 /// near_sdk::env::set_blockchain_interface(Box::new(mocked_blockchain));
 /// ```
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
If we are making a breaking change here, perhaps we should move these public methods elsewhere? 

I think, ideally, we'll have one single public cfg module, so as not to duplicate cfgs

```
#[cfg(not(target_arch = "wasm32"))]
pub mock {
}
```

Reply:
Trying to reduce the amount of breakage for this release. If I were to do this, I would still like to keep this method here and just mark it as deprecated. Do you think this is a good idea?

Honestly seems like there are a lot of breaking changes in next major release and I don't want it to be annoying to migrate.

Reply:
Yeah, that's reasonable, better to keep it here *without* adding a deprecation.

A deprecation can be added in a minor release, and it's nice to be able to switch to new major without seeing a bunch of new deprecations from the get-go.